src/components/InventoryTabs/ImmutableDevices/EdgeDevicesView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
+import { resolveRelPath } from '../../../Utilities/path';
+import {
+  getNotificationProp,
+  manageEdgeInventoryUrlName,
+} from '../../../Utilities/edge';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+const ImmutableDevicesView = (props) => {
+  const dispatch = useDispatch();
+  const notificationProp = getNotificationProp(dispatch);
+  return (
+    <AsyncComponent
+      appName="edge"
+      module="./DevicesView"
```

Comment:
For my knowledge what is the difference between the this fed module and this [one](https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/src/components/InventoryTabs/ImmutableDevices/EdgeDevicesTab.js)

Reply:
DevicesView is the basic component of Inventory without all actions features also inventory does not allow to select items , as it needs that for it's own functionality , we will need to deeply modify Inventory component to allow that , but DevicesView that is a basic component is enough for the current  implementation, that is also used for groups systems selection in edge-frontend